Segment-Taplytics/Classes/SEGTaplyticsIntegration.h

```diff
@@ -20,4 +20,6 @@
 
 - (instancetype)initWithSettings:(NSDictionary *)settings andTaplytics:(id)taplyticsClass;
 
+- (instancetype)initWithoutSettings: andTaplytics:(id)taplyticsClass;
```

Comment:
Surprised this synatx works but how about:

`- (instancetype)initWithTaplytics:(id)taplyticsClass;`